.eslintrc.json

```diff
@@ -1,8 +1,29 @@
 {
+  "env": {
+    "browser": true,
+    "node": true,
+    "jest": true
+  },
   "extends": ["airbnb-base", "plugin:prettier/recommended"],
   "rules": {
+    "no-unused-vars": [
+      1,
+      {
+        "argsIgnorePattern": "res|next|^err|_",
+        "ignoreRestSiblings": true
+      }
+    ],
     "no-use-before-define": 0,
     "semi": 0,
-    "comma-dangle": ["error", "always-multiline"]
+    "comma-dangle": [
+      "error",
+      {
+        "arrays": "always-multiline",
+        "objects": "always-multiline",
+        "imports": "never",
+        "exports": "never",
+        "functions": "never"
+      }
+    ]
```

Comment:
after upgrading eslint and prettier I found it easier to change the settings to be the same as PF, but a lot of changes in code!